topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorCommandLineArgs.java

```diff
@@ -30,5 +30,27 @@ public File getConfig() {
         public String getEntriesCsvFilePath() {
             return entriesCsvFilePath;
         }
+
+        /**
+         * Headers for the input data file of entries to generate AI topics for.
+         */
+        public enum InputCsvHeaders {
+            trsId, version
+        }
+
+        /**
+         * Headers for the output file containing the AI generated topics.
+         */
+        public enum OutputCsvHeaders {
+            trsId,
+            version,
+            descriptorUrl, // Raw GitHub URL of the descriptor file used to generate the topic
+            descriptorChecksum, // Checksum of the descriptor file used to generate the topic. Can be used to determine if the content has changed
+            isTruncated, // Whether the descriptor file content truncated because it exceeded the token maximum
+            promptTokens, // Number of tokens in prompt
+            completionTokens, // Number of tokens in response
+            finishReason, // The reason that the response stopped
+            aiTopic
```

Comment:
These non-CAPITAL enum constants make me feel funny, but I'm ok with that if it's a typical style practice.

Reply:
It's a downside of using enums to define the column headers, but apache commons does warn that it may lead to enum constants that don't follow the standard of constants with all caps https://commons.apache.org/proper/commons-csv/user-guide.html#Defining_a_header_manually